docs/docs/rest-api/public/api/v2/types/volumes.raml

```diff
@@ -6,7 +6,79 @@ uses:
   pragma: pragma.raml
   secrets: secrets.raml
 
+
 types:
+  AppCSIVolume:
+    type: object
+    properties:
+      csi: ExternalCSIVolumeInfo
+  ExternalCSIVolumeInfo:
+    type: object
+    properties:
+      pluginName:
+        type: string
+      provisioning: StaticProvisioning
+  StaticProvisioning:
+    type: object
+    additionalProperties: false
+    properties:
+      static:
+        type: object
+        properties:
+          volumeId:
+            type: string
+            description: The ID of the volume to publish
+          accessType: BlockOptions | MountOptions
```

Comment:
@qianzhangxa @greggomann should this relate to `containerPath`?

Reply:
@pierrebeitz No, it is not related to `containerPath`. `containerPath` is a path inside the container at which the volume will be mounted regardless the type of the volume.